packages/react-components/src/components/ScreenShareButton.tsx

```diff
@@ -67,3 +71,18 @@ export const ScreenShareButton = (props: ScreenShareButtonProps): JSX.Element =>
     />
   );
 };
+
+const screenshareButtonStyles = (theme: Theme): IButtonStyles => ({
+  rootChecked: {
+    background: theme.palette.themePrimary,
+    color: DefaultPalette.white,
```

Comment:
curious about the usage of `DefaultPalette` for color instead of `theme`?

Reply:
I left this as is (this code was just moved from one file to another) but also curious on this too if anyone knows why default palette is used

Reply:
I think @PorterNan worked on this originally.